### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by samccone**

### CODE REVIEW

**Comment by jdaudier** [comment]
yay :+1:

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 language: node_js
 sudo: false
 node_js:
-  - "0.12"
+  - "5.3"
```
Comment (peterblazejewicz) [inline .travis.yml]: @samccone 
Why not 4.2.4 (LTS)?

Comment (samccone) [inline .travis.yml]: heh, we  could switch to 4 :)

Comment (peterblazejewicz) [inline .travis.yml]: Aye sir!

**Pull request merged**